infrastructure/infrastructure.tf

```diff
@@ -79,8 +79,29 @@ module "elastic_resources" {
   ssm_source_stage = "${local.ssm_source_stage}"
 }
 
+module "nmap_task_shared" {
```

Comment:
We don't want to have to import two modules here. We only want to import the ecs_task. The ecs_task should itself use the simpler module